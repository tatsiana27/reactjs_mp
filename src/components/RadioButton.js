import React, { Fragment } from 'react';

import './radio-button.scss';

export function RadioButton(props) {
    return (
        <Fragment>
            <input
                id={props.id}
                type="radio"
                name={props.name}
                value={props.value}
                checked={props.isChecked}
            />
            <label htmlFor={props.name}>{props.value}</label>
        </Fragment>
    );
}