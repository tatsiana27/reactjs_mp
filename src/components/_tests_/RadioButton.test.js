import React from 'react';
import { RadioButton } from '../RadioButton';
import { shallow } from 'enzyme';

describe('RadioButton ', () => {
    test('should render RadioButton Component', () => {
        const wrapper = shallow(
            <RadioButton/>

        );

        expect(wrapper).toMatchSnapshot();
    });
});