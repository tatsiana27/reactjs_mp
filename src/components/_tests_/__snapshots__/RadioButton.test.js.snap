// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton  should render RadioButton Component 1`] = `
<React.Fragment>
  <input
    type="radio"
  />
  <label />
</React.Fragment>
`;
